data/json/items/ammo.json

```
@@ -804,7 +804,7 @@
     "category": "spare_parts",
     "price": 2000,
     "price_postapoc": 100,
-    "name": { "str": "medical tape" },
+    "name": { "str": "medical tape", "str_pl": "medical tape rolls" },
```

```suggestion
    "name": { "str": "medical tape", "str_pl": "rolls of medical tape" },
```
Check to see if you can concatenate like this without the game exploding. I don't think you can.